drivers/gpu/drm/panel/panel-samsung-s6e3fa0.c

```diff
@@ -0,0 +1,741 @@
+{MCS_GAMMA, 0,   222, 0,   228, 0,   219, 132, 131, 132, 133, 132, 131, 132, 132, 131, 132, 133, 133, 130, 130, 129, 130, 133, 130, 138, 143, 135, 157, 141, 150, 0,   0,   0  },
+{MCS_GAMMA, 0,   226, 0,   232, 0,   224, 132, 131, 131, 132, 131, 131, 132, 132, 131, 132, 133, 133, 130, 130, 129, 131, 133, 131, 135, 139, 133, 149, 137, 144, 0,   0,   0  },
+{MCS_GAMMA, 0,   230, 0,   235, 0,   228, 132, 132, 132, 131, 131, 129, 132, 132, 131, 131, 132, 132, 131, 131, 130, 128, 131, 128, 135, 139, 133, 149, 137, 144, 0,   0,   0  },
+{MCS_GAMMA, 0,   234, 0,   238, 0,   232, 131, 131, 131, 130, 130, 130, 131, 132, 131, 130, 131, 131, 127, 129, 128, 131, 133, 132, 131, 137, 128, 145, 135, 141, 0,   0,   0  },
+{MCS_GAMMA, 0,   209, 0,   217, 0,   203, 132, 131, 131, 133, 133, 133, 132, 134, 133, 131, 132, 134, 130, 128, 128, 129, 131, 130, 136, 136, 135, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   213, 0,   221, 0,   208, 132, 130, 131, 133, 132, 132, 132, 134, 133, 130, 132, 133, 131, 129, 129, 131, 132, 131, 131, 133, 132, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   218, 0,   224, 0,   213, 131, 130, 130, 132, 131, 131, 132, 133, 133, 132, 133, 134, 129, 128, 128, 128, 129, 128, 131, 133, 132, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   221, 0,   227, 0,   217, 131, 130, 130, 131, 131, 131, 131, 132, 131, 131, 132, 133, 129, 128, 128, 128, 129, 128, 131, 133, 132, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   225, 0,   230, 0,   221, 131, 130, 130, 130, 130, 130, 131, 132, 131, 128, 129, 130, 129, 128, 128, 128, 129, 128, 131, 133, 132, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   229, 0,   234, 0,   226, 130, 129, 129, 130, 130, 130, 130, 131, 130, 129, 130, 131, 130, 128, 128, 129, 130, 130, 127, 129, 128, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   234, 0,   238, 0,   231, 130, 129, 129, 130, 130, 130, 130, 131, 131, 129, 130, 131, 128, 127, 127, 127, 128, 127, 127, 129, 128, 134, 132, 133, 0,   0,   0  },
+{MCS_GAMMA, 0,   238, 0,   241, 0,   236, 129, 129, 129, 130, 130, 130, 130, 130, 130, 129, 129, 130, 129, 128, 128, 128, 129, 129, 132, 131, 131, 127, 127, 127, 0,   0,   0  },
+{MCS_GAMMA, 0,   244, 0,   246, 0,   242, 128, 128, 128, 129, 129, 129, 130, 130, 130, 128, 129, 129, 129, 129, 129, 130, 130, 130, 127, 127, 127, 127, 127, 127, 0,   0,   0  },
+{MCS_GAMMA, 0,   247, 0,   249, 0,   246, 129, 129, 129, 129, 129, 129, 129, 129, 129, 128, 128, 128, 130, 129, 129, 127, 127, 127, 127, 127, 127, 127, 127, 127, 0,   0,   0  },
+{MCS_GAMMA, 0,   252, 0,   252, 0,   251, 128, 128, 128, 129, 129, 129, 129, 129, 129, 127, 127, 127, 127, 127, 127, 127, 127, 127, 127, 127, 127, 127, 127, 127, 0,   0,   0  },
+{MCS_GAMMA, 1,   0,   1,   0,   1,   0,   128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 128, 0,   0,   0  },
```

Comment:
./scripts/checkpatch.pl should warn about lines in this block being too long?

Reply:
It isn't for me, I even tried with --fix-inplace and it did not do anything to this table.

./scripts/checkpatch.pl -f --fix-inplace --no-tree drivers/gpu/drm/panel/panel-samsung-s6e3fa0.c

The only thing it complains about is a warning on line 248 that the dsi_generic_write_seq macro has flow control statements.  Since this will have to be reworked anyways for 6.0 I assume it's not a warning to worry about now?

Reply:
> It isn't for me

:ok: 

>  it's not a warning to worry about now?

Yes, right